_posts/2020-12-04-a-new-way-to-learn-and-discover-quarkus.adoc

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title: A new way to learn & discover Quarkus (even for the Quarkus veterans)
```

Comment:
the title is very "flat" and you don't really know what it is about until half way into the blog post. 

How about:

```suggestion
title: Codestarts - A new way to learn & discover Quarkus ```

then you have a name and know what is about.